README.md

```diff
@@ -143,22 +143,21 @@ Since the resource state is stored outside of the container, the GPU cluster lif
 ## Instructions
 
 1. gcloud auth application-default login.
-2. gcloud auth configure-docker us-docker.pkg.dev
```

Comment:
This is the line that was removed.

Everything else is just a numbering change.